readiness-tool/README.md

```diff
@@ -0,0 +1,21 @@
+
+# AMP Readiness Chrome Extension
+
+## Introduction: 
+The AMP Readiness Tool (ART) is designed to assist the Sales team in determining whether AMP can be incorporated into a website's existing mobile page. The tool examines, via Chrome Extension, the different services, ads and analytics networks utilized on the page and specifies whether each service currently has AMP support. For the services that aren't supported, workarounds and substitutions will be given in order for the Sales team to better estimate the time and energy required for the website to get "AMPED" up.
```

Comment:
take out "Sales Team" specifics. In reality, anyone will be able to use this. Change to:

"The AMP Readiness Tool (ART) is designed to help determine whether AMP can be incorporated into a website's existing mobile page"

and 

"For the services that aren't supported, workarounds and substitutions will be given in order to better estimate the time and energy required for the website to get "AMPED" up."